client/network/src/service.rs

```diff
@@ -845,12 +845,12 @@ pub struct NetworkWorker<B: BlockT + 'static, H: ExHashT> {
 
 struct Metrics {
 	// This list is ordered alphabetically
-	connections: GaugeVec<U64>,
+	connections_opened_total: CounterVec<U64>,
 	connections_closed_total: CounterVec<U64>,
```

Comment:
```suggestion
	connections_closed_total: CounterVec<U64>,
	connections_opened_total: CounterVec<U64>,
```

Friendly reminder from Pierre from the past that *this list is ordered alphabetically*.